plasmapy/particles/particle_collections.py

```diff
@@ -292,24 +301,37 @@ def is_category(
     @property
     def charge_number(self) -> np.array:
         """
-        An array of the quantized charges of the particles, as
-        multiples of the elementary charge.
+        The charges of the particles in units of the elementary
+        positive charge.
```

Comment:
Isn't the elementary charge e defined to be positive?

```suggestion
         charge.
```